src/Http/Controllers/VoyagerMediaController.php

```diff
@@ -339,34 +347,69 @@ public function crop(Request $request)
 
         $realPath = Storage::disk($this->filesystem)->getDriver()->getAdapter()->getPathPrefix();
         $originImagePath = $realPath.$request->upload_path.'/'.$request->originImageName;
+        $originImagePath = str_replace('\\', '/', $originImagePath);
+        $originImagePath = preg_replace('#/+#', '/', $originImagePath);
+
+        if ($createMode) {
+            // create a new image with the cpopped data
+            $fileNameParts = explode('.', $request->originImageName);
+            array_splice($fileNameParts, count($fileNameParts) - 1, 0, 'cropped_'.time());
+            $newImageName = implode('.', $fileNameParts);
+            $destImagePath = $realPath.$request->upload_path.'/'.$newImageName;
+        } else {
+            // override the original image
+            $destImagePath = $originImagePath;
+        }
+        $destImagePath = str_replace('\\', '/', $destImagePath);
+        $destImagePath = preg_replace('#/+#', '/', $destImagePath);
 
-        try {
-            if ($createMode) {
-                // create a new image with the cpopped data
-                $fileNameParts = explode('.', $request->originImageName);
-                array_splice($fileNameParts, count($fileNameParts) - 1, 0, 'cropped_'.time());
-                $newImageName = implode('.', $fileNameParts);
-                $destImagePath = $realPath.$request->upload_path.'/'.$newImageName;
-            } else {
-                // override the original image
-                $destImagePath = $originImagePath;
-            }
-
-            Image::make($originImagePath)->crop($width, $height, $x, $y)->save($destImagePath);
+        $image = null;
+        $success = false;
 
+        try {
+            $image = Image::make($originImagePath)->crop($width, $height, $x, $y);
+            $image->save($destImagePath);
             $success = true;
             $message = __('voyager::media.success_crop_image');
         } catch (Exception $e) {
-            $success = false;
             $message = $e->getMessage();
         }
 
+        if (!$success) {
+            try {
+                $content = Storage::disk($this->filesystem)->get($originImagePath);
+                $image = Image::make($content)->crop($width, $height, $x, $y);
+                Storage::disk($this->filesystem)->put($destImagePath, (string) $image->encode());
+                $success = true;
+                $message = __('voyager::media.success_crop_image');
+            } catch (Exception $e) {
+                $message = $e->getMessage();
+            }
+        }
+
         return response()->json(compact('success', 'message'));
     }
 
-    private function addWatermarkToImage($image, $options)
+    private function addWatermarkToImage(\Intervention\Image\Image $image, $options)
```

Comment:
`Image` is already imported.  No need to use the full namespace here.

Reply:
fixed in latest commit